modules/nodes-cluster-worker-latency-profiles-using.adoc

```diff
@@ -138,3 +140,79 @@ $ oc get KubeControllerManager -o yaml | grep -i workerlatency -A 5 -B 5
 
 To change the medium profile to default or change the default to medium, edit the `node.config` object and set the `spec.workerLatencyProfile` parameter to the appropriate value.
 
+[id="nodes-cluster-worker-latency-profiles-using-at-creation_{context}"]
+== Using worker latency profiles at cluster creation
+
+To implement a worker latency profile in the manifest, follow these steps:
```

Comment:
Not sure if you saw my comment in the resolved chat below; we need the `openshift-install create manifests` to create the default node manifest yaml (and other files) before we add in the workerLatencyProfile. It needs a specific set up of folders to be able to do the installation properly

Reply:
@paigerube14 You clarified via slack conversation. I'll gladly make this update, thank you so much!

Reply:
@paigerube14 I have added an IMPORTANT box and a reference to the Installing section of the openshift documentation. The IMPORTANT box points out the workerLatencyProfile must be added after the manifest and file structure is created.